src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4541,9 +4544,7 @@ void os::init(void) {
     (int(*)(pthread_t, const char*))dlsym(RTLD_DEFAULT, "pthread_setname_np");
 
   check_pax();
-
-  // Check the availability of MADV_POPULATE_WRITE.
-  FLAG_SET_DEFAULT(UseMadvPopulateWrite, (::madvise(0, 0, MADV_POPULATE_WRITE) == 0));
+  Linux::version_init();
```

Comment:
check_pax() and Linux::version_init() are unrelated things. Please add a blankline between them.